packages/medusa/src/api/routes/store/customers/list-orders.ts

```diff
@@ -1,6 +1,19 @@
 import { Type } from "class-transformer"
-import { IsNumber, IsOptional, IsString } from "class-validator"
+import {
+  IsEnum,
+  IsNumber,
+  IsOptional,
+  IsString,
+  ValidateNested,
+} from "class-validator"
+import _, { identity } from "lodash"
```

Comment:
nit (not much difference really, so take it or leave it 😅 ):
```suggestion
import { identity, omit, pickBy } from "lodash"
```